icu4j/APIChangeReport.html

```diff
@@ -2,233 +2,171 @@
 <html>
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-<!-- © 2021 and later: Unicode, Inc. and others. -->
+<!-- © 2022 and later: Unicode, Inc. and others. -->
 <!-- License & terms of use: http://www.unicode.org/copyright.html -->
-<title>ICU4J API Comparison: ICU4J 69.1 with ICU4J 70.1</title>
+<title>ICU4J API Comparison: ICU4J 70.1 with ICU4J 71.0.1</title>
 </head>
 <body>
-<h1>ICU4J API Comparison: ICU4J 69.1 with ICU4J 70.1</h1>
+<h1>ICU4J API Comparison: ICU4J 70.1 with ICU4J 71.0.1</h1>
 
 <hr/>
-<h2>Removed from ICU4J 69.1</h2>
-
-<h3>Package com.ibm.icu.lang</h3>
-<ul>
-UCharacter
-<ul>
-<li><span style='color:green'>(stable)</span> public static boolean <i>isHighSurrogate</i>(char)</li>
-<li><span style='color:green'>(stable)</span> public static boolean <i>isLowSurrogate</i>(char)</li>
-<li><span style='color:green'>(stable)</span> public static final boolean <i>isSurrogatePair</i>(char, char)</li>
-<li><span style='color:green'>(stable)</span> public static final int <i>toCodePoint</i>(char, char)</li>
-</ul>
-</ul>
-
-<h3>Package com.ibm.icu.text</h3>
-<ul>
-UTF16
-<ul>
-<li><span style='color:green'>(stable)</span> public static boolean <i>isLeadSurrogate</i>(char)</li>
-<li><span style='color:green'>(stable)</span> public static boolean <i>isSurrogate</i>(char)</li>
-<li><span style='color:green'>(stable)</span> public static boolean <i>isTrailSurrogate</i>(char)</li>
-</ul>
-</ul>
-
+<h2>Removed from ICU4J 70.1</h2>
+<p>(no API removed)</p>
 
 <hr/>
-<h2>Deprecated or Obsoleted in ICU4J 70.1</h2>
+<h2>Deprecated or Obsoleted in ICU4J 71.0.1</h2>
```

Comment:
I'm expecting 71.1 at this point. We need ICU4J version updated to 71.1 before generating the change report.

Reply:
Would it be sufficient to have ICU4J 71 without any minor version info?

Reply:
Yes, that's what we said today. Just 70 vs. 71.

Reply:
This should be fixed now.